.circleci/config.yml

```diff
@@ -739,7 +824,7 @@ jobs:
             mkdir -p artifacts/blob-report
             while true; do
               RESPONSE=$(curl -s --request GET "https://circleci.com/api/v2/workflow/$CIRCLE_WORKFLOW_ID/job" --header "Circle-Token: $CCI_Token")
-              FUNCTIONAL_TEST_JOB_ID=$(echo $RESPONSE | jq -r '.items[] | select(.status == "running") | select(.name | startswith("Functional Tests")) | .id')
+              FUNCTIONAL_TEST_JOB_ID=$(echo $RESPONSE | jq -r '.items[] | select(.status == "running") | select(.name | startswith("Firefox Functional Tests")) | .id')
```

Comment:
Not sure if we want to add a comment here on why this needed to be changed (be more specific to account for the Chromium job).

Reply:
Sure, added the comment.